components/browse/LatestSection.vue

```diff
@@ -91,21 +132,15 @@
 
     methods: {
       cardData(card) {
-        let data = {};
-
-        switch (this.category) {
-        case 'Exhibitions':
-          data = this.defaultCardData(card, 'exhibitions-exhibition');
-          break;
-        case 'Galleries':
-          data = this.galleryCardData(card);
-          break;
-        case 'Blog posts':
-          data = this.defaultCardData(card, 'blog-all');
-          break;
+        if (this.forExhibitions) {
+          return this.defaultCardData(card, 'exhibitions-exhibition');
+        } else if (this.forGalleries) {
+          return this.galleryCardData(card);
+        } else if (this.forBlogPosts) {
+          return  this.defaultCardData(card, 'blog-all');
         }
 
-        return data;
+        return null;
```

Comment:
are there any instances where neither exhibitions/galleries/blog posts will be true when calling this and null will then be returned? does this throw an error? is there a test for this?